scripts/zkai_attention_kv_two_head_bounded_softmax_table_native_gate.py

```
@@ -0,0 +1,590 @@
+    module = importlib.util.module_from_spec(spec)
+    try:
+        spec.loader.exec_module(module)
+    except Exception as err:
+        raise ImportError(f"failed to import {module_name} from {path}: {err}") from err
+    return module
+
+
+INPUT_MODULE = load_script_module(INPUT_SCRIPT, "zkai_attention_kv_stwo_native_two_head_bounded_softmax_table_proof_input")
+
+
+def read_bounded_json(path: pathlib.Path, max_bytes: int, label: str) -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `read_bounded_json`

(ANN401)